docs/release-notes/release-notes-0.1.0.md

```diff
@@ -0,0 +1,310 @@
+DCR) at a rate of 0.001207 BTC/DCR using a standing time-in-force to allow the
+order to become booked if it is not filled:
+
+![submit order dialog](https://user-images.githubusercontent.com/9373513/97030709-c8fd9500-1524-11eb-8bd0-3eb4cf95e8c8.png)
+
+Checking the "Match for one epoch only" box above specifies that the limit
+order's time-in-force should be immediate, while unchecking it allows the order
+to be booked if it does not match with another order at first. The concept of
+epochs is described in the [Epoch](#epochs) section.
+
+### Order Funding
+
+Since orders must be funded by coins from the user's wallets, placing an order
+"locks" an amount in the relevant wallet. For example, a buy order on the
+DCR-BTC market marks a certain quantity of BTC as locked with the user's wallet.
+(This involves no transactions or movement of funds.) This amount will be shown
+in the "locked" row of the Balances table.
+
+It is important to note that the amount that is locked by the order may be
+**larger than the order quantity** since the "locked" amount is dependent on the
+size of the UTXO (for UTXO-based assets like Bitcoin and Decred) that is
+reserved for use as an input to the swap transaction, where the amount that does
+not enter the contract goes in a change address. This is no different from when
+you make a regular transaction, however because the input UTXOs are locked in
+advance of broadcasting the actual transaction that spends them, you will see
+the amount locked in the wallet until the swap actually takes place.
+
+Depending on the asset, there may be a wallet setting on the Wallets page to
+pre-size funding UTXOs to avoid this over-locking, but (1) it involves an extra
+transaction that pays to yourself before placing the order, which has on-chain
+transaction fees that may be undesirable on chains like BTC, and (2) it is only
+applied for limit orders with standing time-in-force since the the UTXOs are
+only locked until the swap transaction is broadcasted, which is relatively brief
+for taker-only orders that are never booked.
+
+### Epochs
+
+An important concept with DCRDEX is that newly submitted orders are processed in
+short windows of time called **epochs**, the length of which is part of the
+server's market configuration, but is typically on the order of 10 seconds. When
+a valid order is received by the server, it enters into the pool of epoch orders
+before it is matched and/or booked. The motivation for this approach is
+described in detail in the DCRDEX specification. The Your Orders table will show
+the status of such orders as "epoch" until they are matched at the end of the
+epoch, as described in the next section.
+
+Order cancellation requests are also processed in the epoch with trade
+(market/limit) orders since a cancellation is actually a type of order. However,
+from the user's perspective, cancelling an order is simply a matter clicking the
```

Comment:
Adding the following to the Matching section: "A cancel order may also fail
to match if another trade matches with the targeted order first, or it may match
after the targeted order is partially filled in the same epoch."